sdk/ml/azure-ai-ml/azure/ai/ml/constants/_compute.py

```diff
@@ -3,78 +3,134 @@
 # ---------------------------------------------------------
 
 
-class ComputeType(object):
+class ComputeType:
```

Comment:
Same question here. I thought these files are private interface.

Reply:
All the objects(private/public) should be documented.